features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -222,6 +230,21 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
                 }
             }
         }
+        viewModel.markerClickCallback.observe(viewLifecycleOwner){
+            if (viewModel.exploreTopic == ExploreTopic.Merchants){
+                analyticsService.logEvent(AnalyticsConstants.ExploreDash.SELECT_MERCHANT_MARKER, bundleOf())
+            }
+        }
+        viewModel.zoomLevelChangeCallback.observe(viewLifecycleOwner){
+            if (viewModel.exploreTopic == ExploreTopic.Merchants){
+                analyticsService.logEvent(AnalyticsConstants.ExploreDash.ZOOM_MERCHANT_MAP, bundleOf())
+            }
+        }
+        viewModel.cameraCenterChangeCallback.observe(viewLifecycleOwner){
```

Comment:
`SearchFragment` doesn't have to know anything about the map, I don't get why this is here.